include/jemalloc/internal/tsd_win.h

```diff
@@ -8,10 +8,12 @@
 #include "jemalloc/internal/tsd_types.h"
 
 typedef struct {
-	bool initialized;
 	tsd_t val;
+	bool initialized;
```

Comment:
Unintentional change? For the context, `tsd_t` is rather large and may contain stuff rarely touched at its end (TSD_DATA_SLOWER). So putting the init state at the beginning can avoid touching a remote page.

Reply:
This was intended because accessing initialized is not the common path, while the common path is tsd_get. Without this change, we need to do tsd_wrapper + ptr_size to get the pointer to val. With this change, the + ptr_size operation is not needed anymore. Sure, accessing initialized may touch a remote page, but that's only done when tsd is set up.

Reply:
Makes sense. Can you please add some comments about avoiding the pointer shift to get the val?

Reply:
Sure, done.